src/app/meetups/meetups.component.html

```diff
@@ -70,6 +70,16 @@ <h3 class="header">
         </mat-cell>
       </mat-cell>
     </ng-container>
+    <ng-container matColumnDef="action">
+      <mat-header-cell *matHeaderCellDef i18n> Information </mat-header-cell>
+      <mat-cell *matCellDef="let element" class="list-content-menu">
+        <div>
+          <p>{{"Date: " + (element.startDate | date: 'fullDate') + " - " + (element.endDate | date: 'fullDate') + " at " + element.startTime + ' - ' + element.endTime}}</p>

```

Comment:
@paulbert Should we use translation here ? Since it includes both dynamic and static value .

Reply:
We cannot without a change because it is all in an interpolation ({{...}}) block.  To use translation it needs to be broken up like this:
```
<p><span i18n>Date:</span>{{ ' ' + (element.startDate | date: 'fullDate') + ' - ' + (element.endDate | date: 'fullDate') + ' '}}<span i18n>at</span>{{ ' ' + element.startTime + ' - ' + element.endTime}}</p>
```
Notice that I keep the spaces in the interpolation statements.  Leaving a trailing space would risk the translator remove it for the other languages.

The same change is needed for the next two lines.